Tests/UIKitTests/UIViewExtensionsTests.swift

```diff
@@ -206,6 +206,38 @@ final class UIViewExtensionsTests: XCTestCase {
 		view.addSubviews([UIView(), UIView()])
 		XCTAssertEqual(view.subviews.count, 2)
 	}
+    
+    func testAddSubViewsWithTranslatesAutoresizingMaskIntoConstraints() {
+        let frame = CGRect(x: 0, y: 0, width: 100, height: 100)
+        let view = UIView(frame: frame)
```

Comment:
The frame's irrelevant for these tests, so you can just use `UIView()` like you do later down.
This comment applies to `testAddSubViewWithTranslatesAutoresizingMaskIntoConstraints` as well.